develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -310,55 +469,36 @@ To retrieve the results:
 Example:
 
 ```csharp
-do
+UIBlockDefinition blockItem = new UIBlockDefinition
 {
-     UIBuilder uib = new UIBuilder();
-     uib.Title = "Treeview - default";
-     uib.RequireResponse = true;
-     uib.RowDefs = "*,*";
-     uib.ColumnDefs = "*";
-    
-     UIBlockDefinition tree = new UIBlockDefinition();
-     tree.Type = UIBlockType.TreeView;
-     tree.Row = 0;
-     tree.Column = 0;
-     tree.DestVar = "treevar";
-     tree.TreeViewItems = new List<TreeViewItem>
-     {
-         new TreeViewItem("Core Teams", "1", new List<TreeViewItem>
-         {
-             new TreeViewItem("Team A", "1/1", new List<TreeViewItem>
-             {
-                 new TreeViewItem("Brian", "1/1/1", true) { ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("John", "1/1/2"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Kevin", "1/1/3", true){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("David", "1/1/4"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Joe", "1/1/5", true){ ItemType = TreeViewItem.TreeViewItemType.CheckBox }
-             }){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-             new TreeViewItem("Team B", "1/2", new List<TreeViewItem>
-             {
-                 new TreeViewItem("Jane", "1/2/1"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Sarah", "1/2/2"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Emely", "1/2/3"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Anne", "1/2/4"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-                 new TreeViewItem("Florence", "1/2/5"){ ItemType = TreeViewItem.TreeViewItemType.CheckBox }
-             }) { ItemType = TreeViewItem.TreeViewItemType.CheckBox }
-         }){ ItemType = TreeViewItem.TreeViewItemType.CheckBox },
-         new TreeViewItem("Team C", "2") { ItemType = TreeViewItem.TreeViewItemType.CheckBox }
-     };
-    
-     uib.AppendBlock(tree);
-    
-     UIBlockDefinition next = new UIBlockDefinition();
-     next.Type = UIBlockType.Button;
-     next.Row = 1;
-     next.Column = 0;
-     next.Text = "Next";
-     next.DestVar = "Next";
-     uib.AppendBlock(next);
-    
-     _treeResults = _engine.ShowUI(uib);
-} while (!_treeResults.WasButtonPressed("Next"));
-
-ShowResult();
+  Type = UIBlockType.TreeView,
```

Comment:
Maybe mention the 'using Skyline.DataMiner.Net.AutomationUI.Objects;' statement wich is required for the TreeViewItem.